.tekton/release/bundle-update/pipeline.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: update-bundle-related-images
+spec:
+  params:
+    - name: snapshot
+      type: string
+      description: The component snapshot to update the bundle with. Provided by Konflux.
+    - name: release
+      type: string
+      description: The release name and namespace in the format of <namespace>/<release-name>. Provided by Konflux.
+    - name: releasePlan
+      type: string
+      description: The release plan name, which is not used in this pipeline. Provided by Konflux.
+    - name: forkGitUrl
+      type: string
+      description: The git repository URL where the updated code is committed to.
+      default: "https://github.com/raptorsun/lightspeed-operator.git"
+    - name: forkBranch
+      type: string
+      description: The branch of the git repository where the updated code is committed to.
+      default: "bundle-update"
+    - name: forkUser
+      type: string
+      description: The Github user of the fork git repository.
+      default: "raptorsun"
+    - name: sourceGitUrl
+      type: string
+      description: The source git repository URL to clone as starting point for the bundle update.
+      default: "https://github.com/openshift/lightspeed-operator.git"
+    - name: sourceBranch
+      type: string
+      description: The branch of the source git repository to clone as starting point for the bundle update.
+      default: "main"
+  tasks:
+    - name: update-related-images
+      taskSpec:
+        params:
+          - name: snapshot
+            type: string
+            description: The bundle snapshot to update the catalog with. Provided by Konflux.
+          - name: release
+            type: string
+            description: The release name and namespace in the format of <namespace>/<release-name>. Provided by Konflux.
+          - name: forkGitUrl
+            type: string
+            description: The git repository URL where the updated catalog is committed to
+            default: "https://github.com/raptorsun/lightspeed-operator.git"
+          - name: forkBranch
+            type: string
+            description: The branch of the git repository where the updated catalog is committed to
+            default: "bundle-update"
+          - name: forkUser
+            type: string
+            description: The Github user of the fork git repository
+            default: "raptorsun"
+          - name: sourceGitUrl
+            type: string
+            description: The source git repository URL to clone as starting point for the catalog update
+            default: "https://github.com/openshift/lightspeed-operator.git"
+          - name: sourceBranch
+            type: string
+            description: The branch of the source git repository to clone as starting point for the catalog update
+            default: "main"
+        steps:
+          - name: git-clone
+            image: quay.io/hasun/bundle-update:0.0.1
```

Comment:
shouldn't we have this saved in a repo under openshift lightspeed?

Reply:
I will move to team repo later.